test/test_codegen.cpp

```diff
@@ -177,16 +177,16 @@ TEST(CodeGenTest, RemovedMemberAlignment) {
         Primitive: int8_t
 )",
                 R"(
-[0] Class: MyClass_0 (size: 24, align: 8)
-      Member: a_0 (offset: 0, align: 1)
+[0] Class: MyClass_0 [MyClass] (size: 24, align: 8)
+      Member: a_0 [a] (offset: 0, align: 1)
```

Comment:
This member output from Printer seems particularly unnecessary - it'll only ever be removing the `_{i}` or leaving the dots in vptrs. I erred on the side of printing all of the information in the type graph though.